### ISSUE TITLE
🐛 Sample registration isn't working

### ISSUE THREAD

**Issue by kcullion** [icgc-argo/platform-ui#1191]
## Describe the bug
Sample registration is failing in Production.

## Steps To Reproduce

Steps to reproduce the behaviour:
1. sample registration
2. upload a file
3. register samples
hangs on:
<img width="1568" alt="Screen Shot 2020-03-05 at 2 17 10 PM" src="https://user-images.githubusercontent.com/18424138/76017836-4076a980-5eed-11ea-82db-556abce32ebd.png">

## Expected behaviour
Samples should register

**Comment by kcullion** [comment]
@hlminh2000 it works for me now, thanks!

**Comment by rosibaj** [comment]
tested on production - was able to successfully register data in RAWR-CA
